### ISSUE TITLE
Keep having this crash + sometimes the structure is built wrong

### ISSUE THREAD

**Issue by Mr-Syntax** [TeamMoegMC/TowersOfTheWildReloaded#1]
[crash-2021-11-13_03.03.57-client.txt](https://github.com/TeamMoegMC/TowersOfTheWildReloaded/files/7531235/crash-2021-11-13_03.03.57-client.txt)

it happend after adding dungeons enhanced, but the problem persists even after removing that mod... i have no clue whats going on

**Comment by Mr-Syntax** [comment]
okay the structure is not built wrong that was just me that didnt understand the derelict layout

but this crash is very ellusive to me, i have a very big pack 330 mods or so and it only happens if i use something other than the default preset in terraforged

**Comment by yuesha-yc** [comment]
Yeah, TerraForged is really not very compatible with world gen mods. I don't have an idea.

**Comment by Mr-Syntax** [comment]
sad :-/ well ill let you know if i narrow it down

**Comment by 3inya** [comment]
im having the same issue

**Comment by skeletor261** [comment]
This is the same issue I have reported (https://github.com/TeamMoegMC/TowersOfTheWildReloaded/issues/2) and identified where the problem lies. Towers of the wild is trying to call functions from the Chunk Generator which in this case is created by TF. The functions don't appear in TF code base so likely this was an assumption based on the default. The reason this is crashing is because as I've shown on issue 2 there is no exception handling happening.
